interface/wx/splitter.h

```diff
@@ -68,6 +68,13 @@ enum
         May be used to modify the position of the tracking bar to properly
         reflect the position that would be set if the drag were to be completed
         at this point. Processes a @c wxEVT_SPLITTER_SASH_POS_CHANGING event.
+    @event{EVT_SPLITTER_SASH_POS_RESIZE(id, func)}
+        The sash position is in the process of being updated.
+        May be used to modify the position of the tracking bar to properly
+        reflect the position that would be set if the update were to be completed.
+        This can happen i.E. when the window is resized and the sash is moved
+        according to the gravity setting.
```

Comment:
I'm not sure whether "this can happen" is really the right thing to say here, IMO it would be 
```suggestion
        This even is sent when the window is resized and allows the application to select
        the desired new sash position. If it doesn't process the event, the position
        is determined by the gravity setting.
```

We also need to add "This event is available since wxWidgets 3.1.6.".